ci/scripts/get-bitstream-strategy.sh

```diff
@@ -43,23 +44,61 @@ bitstream_commit=$(ci/bazelisk.sh run //util/py/scripts:get_bitstream_build_id \
   --schema ${REPO_TOP}/rules/scripts/bitstreams_manifest.schema.json \
   --design ${bitstream_design})
 
+bazel_to_path() {
+  # Loop over bazel label dependencies and turn them into paths
+  for dep in $1; do
+    # Ignore everything that does not start with "//"
```

Comment:
To be on the safe side, I would still allow "@//" (which is the same as //) we can be returned by bazel in some circumstances (and I think in newer versions of bazel). Otherwise the filtering could break by mistake